Source/Core/Component/ComponentBase.swift

```diff
@@ -39,9 +54,20 @@ open class ComponentBase<STATE, ACTION>: ComponentWithDelegate {
         componentDelegate.canUpdate = canUpdate
     }
 
+    /**
+    After overriding this function, it can be used for additional setting up independently of **init**.
+
+    As the name implies, this function is called after **init**.
+    */
     open func afterInit() {
```

Comment:
Subscribe all observables that you need to be subscribed to only once, or if the subscription is meant to last the whole existence of the Component.